src/components/Contacts/ContactsMap/ContactsMapPanel.tsx

```diff
@@ -118,7 +118,7 @@ export const ContactsMapPanel: React.FC<ContactMapsPanelProps> = ({
     .map(([statusKey, status]) => {
       return {
         key: statusKey,
-        title: status.translated,
+        title: status.name,
```

Comment:
Why not `status.translated`?

Reply:
Both `status.translated` and `status.name` are the same for normal statuses. But Filter Contact Statuses have
different values for each.
`    NULL: {
      name: 'null',
      translated: t('-- None --'),
      phase: null,
    },
    ACTIVE: {
      name: 'active',
      translated: t('-- All Active --'),
      phase: null,
    },
    HIDDEN: {
      name: 'hidden',
      translated: t('-- All Hidden --'),
      phase: null,
    },`

Reply:
That makes sense